Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -304,9 +304,23 @@ public function clearChanged()
      */
     public function clearEmptyTokens()
     {
-        $count = 0;
+        $limit = $this->count();
+        $index = 0;
+
+        for ($index = 0; $index < $limit; ++$index) {
```

Comment:
``$index=0`` is not necessary here as you initialize it above (ot the above initialization is useless)